wavefront_opentracing_sdk/span.py

```diff
@@ -92,6 +94,27 @@ def set_tag(self, key, value):
                         force_sampling)
         return self
 
+    def log_kv(self, key_values, timestamp=None):
+        """Add a log record to the span.
+
+        :param key_values: A dict of string keys and values of any type can be
+        transferred to string by str()
+        :type key_values: dict
+        :param timestamp: A unix timestamp per :meth:`time.time()`; current
+            time if ``None``
+        :type timestamp: float
+        :return: span itself
+        :rtype: WavefrontSpan
+        """
+        if key_values:
+            fields = {}
```

Comment:
If `key_values` is a `dict`, you may do a dict comprehension here, like
`fields = {k: str(v) for k, v in key_values.items()}`